tools/sdk-readme-table-generator/TableGenerator/Program.fs

```diff
@@ -8,19 +8,19 @@ open TableGenerator.Table
 let inputBranches =
         [ { GitBranchName = "master"
             DisplayName = "Master<br>(6.0.x&nbsp;Runtime)"
-            AkaMsChannel = Some("net6/dev") }
+            AkaMsChannel = Some("6.0/daily") }
           { GitBranchName = "release/6.0.1xx-preview1"
             DisplayName = "Release/6.0.1xx-preview1<br>(6.0.x&nbsp;Runtime)"
-            AkaMsChannel = Some("net6/preview1") }
+            AkaMsChannel = Some("6.0-preview1/daily") }
```

Comment:
I get "not found" issues when using URLs with this pattern.  See https://aka.ms/dotnet/6.0-preview1/daily/Sdk/dotnet-sdk-win-x64.exe

Reply:
This one should exist, but if no builds have published on that channel since that changed it, it wouldn’t yet. I will check

Reply:
Looks like what happened here is that the last build out of installer on this channel is from 2/9 and the change to update this channel id came in on 2/10. I'll leave this one alone until a fresh build is run.